packages/react-components/react-field/src/components/Field/Field.types.ts

```diff
@@ -67,10 +62,14 @@ export type FieldProps = Omit<ComponentProps<FieldSlots>, 'children'> & {
   /**
    * The `validationState` affects the color of the `validationMessage`, the `validationMessageIcon`
    *
-   * Setting `validationState` to `error` will also set `aria-invalid` to `true` on the Field's child,
-   * `role="alert"` on the `validationMessage`, and for some field components, causes the border to become red.
+   * When `validationState` is `error`:
+   * * The `validationMessage` text is red and has `role="alert"` to announce the message to screen readers
+   * * The Field's child has `aria-invalid="true"`
+   * * Some field components (such as `Input`) have a red border
+   *
+   * @default error (when validationMessage is set)
```

Comment:
Not sure how to best communicate this but looking at the code it's really something like

```suggestion
   * @default error when `validationMessage` is set, neutral otherwise.
```